### DISCUSSION TITLE
Text vertical trim support

### DISCUSSION THREAD

**Discussion by Ahmad-Fahmi** [six7/figma-tokens#1769] [Ideas]
Hello Jan, the latest LBU from Figma has a new cool feature to trim text box vertical blank spaces which is nice but I can't control it from within the plugin, so just wondering if we can have it soon.
,Thanks Jan

**Comment by six7** [comment]
Unfortunately there's no support for this for Figma's Plugin API yet. You can keep up to date with that here. https://www.figma.com/plugin-docs/updates/

Should Figma expose this, we might consider adding this, but until then we just can't implement this.

**Reply by Ahmad-Fahmi** [reply]
Thank you Jan ☺️
